kakao/src/main/kotlin/com/agoda/kakao/configurator/KakaoConfigurator.kt

```diff
@@ -0,0 +1,65 @@
+package com.agoda.kakao.configurator
+
+import android.support.test.espresso.DataInteraction
+import android.support.test.espresso.ViewInteraction
+import android.support.test.espresso.web.sugar.Web
+import com.agoda.kakao.delegates.DataInteractionDelegate
+import com.agoda.kakao.delegates.ViewInteractionDelegate
+import com.agoda.kakao.delegates.WebInteractionDelegate
+
+/**
+ * Holds interaction delegates factories used in Kakao.
+ * Interaction delegates are middleware between Kakao and Espresso (*Interaction classes).
+ * By those delegates you can modify common Kakao's API behaviour like to put logs, to handle flaky occurred because of Espresso bugs and others.
+ * You can change factories at any time of a test but we strongly recommend to change before your tests start.
+ */
+object KakaoConfigurator {
```

Comment:
Because it's easier and more obvious to use one point to configure. In our case this point is KakaoConfigurator. So a developer doesn't need to search all config properties through the entire project.

Reply:
In that case it would be nice to have some pretty DSL around it, I think. The whole Kakao is build around the idea of DSL. In this case it would be something like:
```kotlin
kakao {
    viewInteractionFactory = Factory1()
    dataInteractionFactory = Factory2()
    webInteractionFactory = Factory3()
}
```